app/src/main/java/com/example/wearit/WearItApp.kt

```diff
@@ -75,9 +80,27 @@ fun WearItApp() {
                 },
                 setActiveInactive = { viewModel.setItemActiveInactive(it) },
                 currentCategory = uiState.currentCategory,
-                deleteItem = { viewModel.deleteItem(it) }
+                deleteItem = { viewModel.deleteItem(it) },
+                navController = navController,
```

Comment:
Nie przekazuj tu navControllera, tylko funkcję w stylu goToSingleItem, którą będziesz wywoływał po przytrzymaniu jakiegoś itemka (podobnie jak w przypadku na przykład goToPickerScreen). Czyli w tej linijce, zamiast przekazywania navControllera, byłoby coś w stylu:

goToSingleItem = { itemId -> navController.navigate(WearItScreen.ItemInfo.name+"/${itemId}")}

I w garderobie musisz przekazać tę funkcję aż do pojedynczego itemu i tam ją wywoływać po przytrzymaniu z argumentem, którym będzie id itemu

Uzasadnienie:
1. Obecnie całą nawigację mamy w WearItApp.kt i nigdzie indziej nie przekazujemy navControllera, więc traktowanie tego jednego widoku inaczej niepotrzebnie sprawia ze kod jest mniej spójny
2. W przypadku jeśli chcielibyśmy zmienić bibliotekę do nawigacji, to teraz będziemy musieli wprowadzać zmiany również w garderobie, a jeśli w garderobie będziemy jedynie wywoływać funkcje, która triggeruje nawigację to w garderobie nie trzeba będzie nic zmieniać. Inaczej mówiąc garderobę kompletnie nie powinno interesować w jaki sposób działa u nas nawigacja
3. Nie będziemy niepotrzebnie zwiększać poziomu skomplikowania kodu w garderobie, umieszczając tam kod sterujący mechaniką nawigacji